src/Interpreters/DictionaryJoinAdapter.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <Core/Block.h>
+#include <Columns/ColumnVector.h>
+#include <Dictionaries/IDictionary.h>
+#include <Storages/IKVStorage.h>
```

Comment:
Please rename IKVStorage.h into IKeyValueStorage.h.

Reply:
Now it's IKeyValueEntity (because it should not be Storage, just thin Interface that can be applied to any class)